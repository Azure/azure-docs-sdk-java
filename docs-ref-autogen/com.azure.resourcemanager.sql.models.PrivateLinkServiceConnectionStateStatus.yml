### YamlMime:JavaType
uid: "com.azure.resourcemanager.sql.models.PrivateLinkServiceConnectionStateStatus"
fullName: "com.azure.resourcemanager.sql.models.PrivateLinkServiceConnectionStateStatus"
name: "PrivateLinkServiceConnectionStateStatus"
nameWithType: "PrivateLinkServiceConnectionStateStatus"
summary: "Defines values for PrivateLinkServiceConnectionStateStatus."
inheritances:
- "<xref href=\"java.lang.Object\" data-throw-if-not-resolved=\"False\" />"
- "<xref href=\"com.azure.core.util.ExpandableStringEnum\" data-throw-if-not-resolved=\"False\" />"
inheritedMembers:
- "com.azure.core.util.ExpandableStringEnum.<T>fromString(java.lang.String,java.lang.Class<T>)"
- "com.azure.core.util.ExpandableStringEnum.<T>values(java.lang.Class<T>)"
- "com.azure.core.util.ExpandableStringEnum.equals(java.lang.Object)"
- "com.azure.core.util.ExpandableStringEnum.hashCode()"
- "com.azure.core.util.ExpandableStringEnum.toString()"
- "java.lang.Object.clone()"
- "java.lang.Object.finalize()"
- "java.lang.Object.getClass()"
- "java.lang.Object.notify()"
- "java.lang.Object.notifyAll()"
- "java.lang.Object.wait()"
- "java.lang.Object.wait(long)"
- "java.lang.Object.wait(long,int)"
syntax: "public final class PrivateLinkServiceConnectionStateStatus extends ExpandableStringEnum<PrivateLinkServiceConnectionStateStatus>"
constructors:
- "com.azure.resourcemanager.sql.models.PrivateLinkServiceConnectionStateStatus.PrivateLinkServiceConnectionStateStatus()"
fields:
- "com.azure.resourcemanager.sql.models.PrivateLinkServiceConnectionStateStatus.APPROVED"
- "com.azure.resourcemanager.sql.models.PrivateLinkServiceConnectionStateStatus.DISCONNECTED"
- "com.azure.resourcemanager.sql.models.PrivateLinkServiceConnectionStateStatus.PENDING"
- "com.azure.resourcemanager.sql.models.PrivateLinkServiceConnectionStateStatus.REJECTED"
methods:
- "com.azure.resourcemanager.sql.models.PrivateLinkServiceConnectionStateStatus.fromString(java.lang.String)"
- "com.azure.resourcemanager.sql.models.PrivateLinkServiceConnectionStateStatus.values()"
type: "class"
metadata: {}
package: "com.azure.resourcemanager.sql.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-sql:2.1.0
