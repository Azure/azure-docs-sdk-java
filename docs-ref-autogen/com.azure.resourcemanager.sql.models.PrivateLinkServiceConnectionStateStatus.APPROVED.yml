### YamlMime:JavaMember
uid: "com.azure.resourcemanager.sql.models.PrivateLinkServiceConnectionStateStatus.APPROVED*"
fullName: "com.azure.resourcemanager.sql.models.PrivateLinkServiceConnectionStateStatus.APPROVED"
name: "APPROVED"
nameWithType: "PrivateLinkServiceConnectionStateStatus.APPROVED"
members:
- uid: "com.azure.resourcemanager.sql.models.PrivateLinkServiceConnectionStateStatus.APPROVED"
  fullName: "com.azure.resourcemanager.sql.models.PrivateLinkServiceConnectionStateStatus.APPROVED"
  name: "APPROVED"
  nameWithType: "PrivateLinkServiceConnectionStateStatus.APPROVED"
  summary: "Static value Approved for PrivateLinkServiceConnectionStateStatus."
  syntax: "public static final PrivateLinkServiceConnectionStateStatus APPROVED"
type: "field"
metadata: {}
package: "com.azure.resourcemanager.sql.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-sql:2.1.0
